lib/ui/common/theme/theme.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+final ThemeData newsTheme = _buildNewsTheme();
+
+ThemeData _buildNewsTheme() {
+  final base = ThemeData.light();
+
+  return base.copyWith(
+    colorScheme: const ColorScheme.light(),
+    textTheme: _buildNewsTextTheme(base.textTheme),
+  );
+}
+
+TextTheme _buildNewsTextTheme(TextTheme base) {
```

Comment:
dobrze rozumiem, że idea tych zmian jest taka, żeby potem przy wywołaniu np `Theme.of(context).textTheme.bodyText1` było to już zmienione zgodnie z tym co tutaj?

Reply:
Dokładnie, z tego co poprzeglądałem projekty to właśnie takie rozwiązanie wydaje się tym legitnym :D

Reply:
jestem tylko ciekawy na ile będziemy w stanie reużyć tych styli, a ile i tak będziemy musieli potem nadpisywać, ale pomysł spoko

Reply:
No to wyjdzie w praniu i zależy od tego jak bardzo projekt będzie spójny/niespójny - myślę, że może to służyć jako sugestia i jak zobaczymy, że coś jest prawie takie ale nie takie to może stwierdzimy że gwoli spójności lepiej brać styl z theme